CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to Flynt
+
+Thank you for taking the time to contribute! We'd love for you to contribute to our source code and make Flynt even better! Here are the guidelines we'd like you to follow:
+
+- [Code of Conduct](#code-of-conduct)
+- [Submission Guidelines](#submission-guidelines)
+    - [Issues and Bugs](#issues-and-bugs)
+    - [Feature Requests](#feature-requests)
+    - [Pull Requests](#pull-requests)
+- [Coding Rules](#coding-rules)
+    - [Commit Messages](#commit-messages)
+    - [PHP](#php)
+    - [JavaScript](#javascript)
+    - [Templates and Styles](#templates-and-styles)
+
+## Code of Conduct
+Help us keep Flynt open and inclusive. Please read and follow our [Code of Conduct](https://github.com/flyntwp/guidelines/blog/master/CODE_OF_CONDUCT.md).
+
+## Submission Guidelines
+We use GitHub issues for each Flynt repository to keep track of all incoming queries (feature requests, bug reports, and questions). Please use the most relevant repository for submitting your issue:
+
+- [Flynt Theme Issues](https://github.com/flyntwp/flynt-starter-theme/issues)
+- [Flynt Core Issues](https://github.com/flyntwp/flynt-core/issues)
+- [Flynt Docs Issues](https://github.com/flyntwp/flynt-docs/issues)
+- [Flynt CLI Issues](https://github.com/flyntwp/flynt-cli/issues)
+- [Flynt Generator Issues](https://github.com/flyntwp/generator-flynt/issues)
+- [Flynt Docker Issues](https://github.com/flyntwp/docker-flynt-build/issues)
+- [ACF Field Composer Issues](https://github.com/flyntwp/acf-field-group-composer/issues)
+
+### Issues and Bugs
+Before you submit your issue please search GitHub issues first, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
+
+- **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps.
```

Comment:
comma: "if an error is being thrown, a non-minified stack trace helps." also, should these begin with uppercase? (since they're technically sentences)